### ISSUE TITLE
Installation with npm fails ENOENT

### ISSUE THREAD

**Issue by tagru** [microsoft/process-migrator#48]
I am trying to install process-migrator with `npm install process migrator -g` but it always fails with this message. Any ideas how I can solve this?

npm: '6.14.5' 
node: '14.3.0'
OS: Linux 5.6.16-1-MANJARO

```
npm install process-migrator -g
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /home/tagru/.npm-global/lib/node_modules/process-migrator/build/nodejs/nodejs/main.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/tagru/.npm-global/lib/node_modules/process-migrator/build/nodejs/nodejs/main.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/tagru/.npm/_logs/2020-07-03T16_44_01_423Z-debug.log
```

**Comment by AminTi** [comment]
We will release an update addressing previous problems. For new issues, create a ticket, and we'll resolve it promptly. Thank you for you.

**Comment by MOlausson** [comment]
We've located the source of this issue and will release a fix for it shortly. Reopening for traceability.

**Comment by MOlausson** [comment]
Finally released a fix for this with release 0.9.23.
